src/homework9/workingWithLists.java

```diff
@@ -0,0 +1,73 @@
+package homework9;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class workingWithLists {
+
+
+    public static void findOccurance(ArrayList<String> list) {
+//Найбільш схожа структура на JSON, яку вдалось нагуглити
+        HashMap<String, Integer> words = new HashMap<>();
+
+        for (var i = 0; i<list.size(); i++) {
+            if (words.containsKey(list.get(i)) == false) {
+                words.put(list.get(i), 1);
+            } else {
+                words.put(list.get(i), words.get(list.get(i)) + 1);
+            }
+        }
+        System.out.println("findOccurance(): for List "+list);
+        System.out.println(words);
+    }
+
+    public static void calcOccurance(ArrayList<String> list) {
+        ArrayList<String> result = new ArrayList<String>();
+        ArrayList<Integer> count = new ArrayList<>();
+        for (var i = 0; i<list.size(); i++) {
+            if (result.indexOf(list.get(i)) == -1) {
+                result.add(list.get(i));
+                count.add(1);
+            } else {
+                count.set(result.indexOf(list.get(i)), count.get(result.indexOf(list.get(i)))+1);
+            }
+        }
+        System.out.println("calcOccurance(): for List "+list);
+        for (var i = 0; i<result.size(); i++) {
+            System.out.println(result.get(i)+" - "+count.get(i));
+        }
+    }
+
+    public static ArrayList findUnique(ArrayList<Integer> numbers) {
```

Comment:
Maybe it is better to use `int counter = 0` and `counter++` instead of List to calculate how many unique elements? 

Also Set could be used as "Collections that hold only unique elements".